src/permissions.js

```diff
@@ -4,8 +4,8 @@ import { addressesEqual } from './web3-utils'
 // Note that these two terms are slightly confusing artifacts of the ACL:
```

Comment:
Not sure if this is something to worry about but by changing a char in the url of an app, I broke the whole thing and got `Cannot read property 'proxyAddress' of undefined`, maybe we need some whitelist/validation but I don't see this happening too often.

Reply:
Hmm yes, good point. We should go back to the home screen if the proxy address isn't an app.

We may also want to clean the URL but that might be a bit more involved right now.

Reply:
> We may also want to clean the URL but that might be a bit more involved right now.

Let’s do this until we have a proper error message, but ideally I’d avoid it: staying on the entered URL is useful to inform the user about what happened, and to let them correct, if it was a typo (like for 404s).

Reply:
I can’t reproduce (it just displays the home), could you give me an example of app address?

Reply:
@AquiGorka Do you still have the issue with the [path changes](https://github.com/aragon/aragon/pull/899/commits/60b9314803d167fe584656af680ea02a0d6049b8)?